frontend/src/components/multi-step-forms/instance/steps/DuplicationStep.vue

```diff
@@ -222,6 +225,9 @@ export default {
     methods: {
         async getInstanceTypes () {
             const instanceTypes = await instanceTypesApi.getInstanceTypes()
+            const { decorateInstanceTypes } = useInstanceFormHelper()
+
+            decorateInstanceTypes(instanceTypes.types ?? [])
```

Comment:
You both modify the input variable, and return a copy of it, which makes this code mis-leading. In duplication you just run the function, in the Instance step, you create a new assignment. We need to pick a lane (my preference is generally to return a new value, rather than modify the input)